### PR TITLE
Add epoch parser test cases and parseSince method

### PR DESCRIPTION
**Pull request by duanyytop**

### CODE REVIEW

File: ckb/src/main/java/org/nervos/ckb/utils/EpochParser.java [hunk 1]
```diff
@@ -14,6 +14,11 @@ public static EpochParams parse(String hexEpoch) {
   }
 
   public static String parse(long length, long index, long number) {
```
Comment (shaojunda) [inline ckb/src/main/java/org/nervos/ckb/utils/EpochParser.java]: this seems to be generating epoch not parse epoch?

Comment (duanyytop) [reply ckb/src/main/java/org/nervos/ckb/utils/EpochParser.java]: It's interesting and I am not sure `generate` is fine, so I use `parse` to handle. 
I rename it to `generate`.

Comment (duanyytop) [reply ckb/src/main/java/org/nervos/ckb/utils/EpochParser.java]: Updated. https://github.com/nervosnetwork/ckb-sdk-java/pull/293/commits/ca45791f6ce1bfb72fe37dd149750fc14311c7b7

Comment (shaojunda) [reply ckb/src/main/java/org/nervos/ckb/utils/EpochParser.java]: according to https://github.com/nervosnetwork/ckb/pull/1585, I think it's more appropriate to call `generate`

**Pull request merged**